### PR TITLE
Improve logger

### PR DESCRIPTION
**Pull request by wtrocki**

### CODE REVIEW

File: docs/workforce-management-framework/topics/con-logging.adoc [hunk 1]
```diff
@@ -1,10 +1,16 @@
 [id='{context}-con-logging']
 
-*{PROJECT_NAME} Logging* is a facade for link:https://nodejs.org[Node.js] based applications and it is used by all {PROJECT_NAME} modules.
-*{PROJECT_NAME} Logging* is contained in the `{PROJECT_NAME}-core` module.
+*{PROJECT_NAME} Logger* creates logging facade for JavaScript based applications that
+are used by all RainCatcher modules. Developers can use their own loggers by wrapping their implementation into the provided logger interface.
```
Comment (austincunningham) [inline docs/workforce-management-framework/topics/con-logging.adoc]: ```is used by all RainCatcher modules.```

**Review by austincunningham — CHANGES_REQUESTED** [review]

File: docs/workforce-management-framework/topics/con-logging.adoc [hunk 2]
```diff
@@ -1,10 +1,16 @@
 [id='{context}-con-logging']
 
-*{PROJECT_NAME} Logging* is a facade for link:https://nodejs.org[Node.js] based applications and it is used by all {PROJECT_NAME} modules.
-*{PROJECT_NAME} Logging* is contained in the `{PROJECT_NAME}-core` module.
+*{PROJECT_NAME} Logger* creates logging facade for JavaScript based applications that
+is used by all RainCatcher modules. Developers can use their own loggers by wrapping their implementation into the provided logger interface.
```
Comment (witmicko) [inline docs/workforce-management-framework/topics/con-logging.adoc]: >that
 -*{PROJECT_NAME} Logging* is contained in the `{PROJECT_NAME}-core` module.		 +is used by all RainCatcher modules.

suggestion: '... aplications, it is used by all RAin..'

**Review by austincunningham — APPROVED** [review]

File: docs/workforce-management-framework/topics/con-logging.adoc (hunk 2 above)
Comment (witmicko) [reply docs/workforce-management-framework/topics/con-logging.adoc]: >Developers can use their own loggers by wrapping their implementation into the provided logger interface.

Developers can use custom loggers by wrapping their implementation into the provided logger interface.

"their" is repeated twice in one sentence, reads bit awkward.

**Review by witmicko — COMMENTED** [review]
looks good, small suggestions.

File: docs/workforce-management-framework/topics/con-logging.adoc [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [id='{context}-con-logging']
 
 *{PROJECT_NAME} Logger* creates logging facade for JavaScript based applications that
-is used by all RainCatcher modules. Developers can use their own loggers by wrapping their implementation into the provided logger interface.
+is used by all {PROJECT_NAME} modules. Developers can use custom loggers by wrapping their implementation into the provided logger interface..
```
Comment (witmicko) [inline docs/workforce-management-framework/topics/con-logging.adoc]: Doubly terminated sentence

Comment (wtrocki) [reply docs/workforce-management-framework/topics/con-logging.adoc]: Fixed in another PR. Thanks

**Pull request merged**